internal/command/run.go

```diff
@@ -122,6 +126,7 @@ func (c *runCmd) run(cmd *cobra.Command, args []string) {
 	defer c.storage.Close()
 
 	c.uploadRoutinePool = routines.NewPool(1) // run 1 upload in parallel with builds
+	c.buildRoutinePool = routines.NewPool(c.buildWorkers)
```

Comment:
in the master branch "builds" are replaced with the concept of tasks, the pool could be named `taskRunnerRoutinePool` instead